ui/components/app/asset-list/asset-list.js

```diff
@@ -11,10 +11,10 @@ import { useMetricEvent } from '../../../hooks/useMetricEvent';
 import { useUserPreferencedCurrency } from '../../../hooks/useUserPreferencedCurrency';
 import {
   getCurrentAccountWithSendEtherInfo,
-  getNativeCurrency,
   getShouldShowFiat,
   getNativeCurrencyImage,
```

Comment:
Would we also want to pull this one out as well?

Reply:
This one is defined in the base selectors file, which isn't really any better but at least its not in `selectors/send` -- I think long term all selectors should move into appropriate slices to more clearly indicate which part of state they are "selecting" from. For now though we can leave it where it's at.